packages/server/src/api/muster/muster.controller.ts

```diff
@@ -231,30 +233,76 @@ class MusterController {
 
   /**
    * This method returns the normalized rate (0 - 1.0) of compliance on a given
-   * date for a given unit, against  of the unit's muster configs.
+   * date for a given unit, against all of the unit's muster configs.
    */
   async getMusterComplianceByDate(
```

Comment:
This module has 400 LOC already and I wonder if there is a way to split it to smaller modules, if you feel like splitting it.

Reply:
This is valid concern, I think one thing that is a little odd with patterns that were used for the API originally, is we do not separate "business logic" into services (i.e. models, services, controllers). This might help a little bit at least.

Reply:
The list of imports is out of control. While this is a React "single page" component, quickly looking into the imports, I don't see imports of sub-components other than very low level ones. I would expect bunch of higher level components imported.

Reply:
I think this comment is misplaced...you mean the page rather than the controller?

Reply:
Yes, the last one was meant for the impacted React component.